frontend/packages/core/src/card.tsx

```diff
@@ -36,23 +38,71 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  ".MuiGrid-item": {
+    padding: "0px 8px 0px 8px",
+  },
+});
+
+// TODO: use material ui avatar component and implement figma design
+const StyledCardHeaderAvatar = styled.div({
+  width: "32px",
+  height: "32px",
+  marginRight: "16px",
+});
+
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
+  alignSelf: "center",
+});
+
+const StyledGridItem = styled(Grid)({
+  textAlign: "center",
+});
+
+export interface CardHeaderSummaryProps {
+  title: React.ReactNode;
+  subheader?: string;
+}
+
+interface CardHeaderProps {
+  avatar: React.ReactNode;
   children?: React.ReactNode;
+  summary?: CardHeaderSummaryProps[];
+  title: React.ReactNode;
 }
 
-const CardHeader = ({ avatar, children, title }: CardHeaderProps) => (
-  <StyledCardHeaderContainer>
-    <MuiCardHeader
-      style={{
-        padding: "16px",
-      }}
-      disableTypography
-      avatar={avatar}
-      title={<Typography variant="h3">{title}</Typography>}
-    />
-    {children}
-  </StyledCardHeaderContainer>
-);
+const CardHeader = ({ avatar, children, title, summary = [] }: CardHeaderProps) => {
+  return (
+    <StyledCardHeaderContainer>
+      <StyledCardHeader container wrap="nowrap" alignItems="center">
+        <StyledCardHeaderAvatar>
+          <Typography variant="h2">{avatar}</Typography>
+        </StyledCardHeaderAvatar>
+        <Grid item xs>
+          <Typography variant="h4">{title}</Typography>
+        </Grid>
+        {summary.map(section => (
+          <>
+            <StyledDivider orientation="vertical" />
+            <StyledGridItem item xs>
+              {section.title}
+              {section.subheader && (
+                <Typography variant="body3" color="rgba(13, 16, 48, 0.6)">
```

Comment:
this might be another discrepancy in the designs but some have this as body4

Reply:
left a TODO and a comment on figma